GameState.py

```diff
@@ -1,35 +1,46 @@
 from Hand import Hand
 from Player import Player
 
+
 class GameState:
     def __init__(self, discarded: Hand, players: List[Player], current: List[Hand], currentPlayerIndex = -1,lastPlayerIndex = -1):
         self.discarded = discarded 
         self.players = players
         self.current = current # list of hands
-        self.lastPlayerIndex = currentPlayerIndex # last player who puts down non empty Hand, 
-        self.currentPlayerIndex = lastPlayerIndex # current player index to make an action
+        self.lastPlayerIndex = lastPlayerIndex # last player who puts down non empty Hand, 
+        self.currentPlayerIndex = currentPlayerIndex # current player index to make an action
 
-    
+    # determine the next player to make action: using currentPlayerIndex
     def toMove(self):
```

Comment:
I was thinking that we could have `toMove` just return `currentPlayerIndex` (so that `currentPlayerIndex` matches whose turn it is for this gamestate). If we wanted, we could even simplify it to have `currentPlayerIndex` initialized to 0 instead of -1 (just let the first player go regardless of role when the game starts).

Reply:
`currentPlayerIndex` is not actually the current player; it is the last player who has played cards. And it is the rule of the game that landlord always starts first

Reply:
If we make the landlord always the first player when the game is initialized (currently what happens) then the player at index 0 should be the landlord